packages/dd-trace/src/appsec/index.js

```diff
@@ -55,6 +55,7 @@ function enableFromRules (_config, rules) {
   Gateway.manager.addresses.add(addresses.HTTP_INCOMING_ENDPOINT)
   Gateway.manager.addresses.add(addresses.HTTP_INCOMING_RESPONSE_HEADERS)
   Gateway.manager.addresses.add(addresses.HTTP_INCOMING_REMOTE_IP)
+  Gateway.manager.addresses.add(addresses.USER_ID)
```

Comment:
are you sure we need this ? we don't use it for client_ip for example. If there is no rules for the user, then it shouldn't be collected right ?

Reply:
yeah, you're right.